### PR TITLE
chore: fixes anonymous id is preserved in session

### PR DESCRIPTION
**Pull request by jonaslagoni**
**Description**
This PR fixes that if two schemas are simplified in the same session the same counter is used throughout.

### CODE REVIEW

File: src/simplification/Simplifier.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
+
+import { CommonModel, Schema } from '../models';
+import simplifyProperties from './SimplifyProperties';
+import simplifyEnums from './SimplifyEnums';
+import simplifyTypes from './SimplifyTypes';
+import simplifyItems from './SimplifyItems';
```
Comment (magicmatatjahu) [inline src/simplification/Simplifier.ts:6]: Maybe you should add those function inside Simplifier as static methods?

Comment (jonaslagoni) [reply src/simplification/Simplifier.ts:6]: I would rather have them separated completely tbh instead of everything inside one class 😄

Comment (magicmatatjahu) [reply src/simplification/Simplifier.ts:6]: Do you use in other place these function? I think you can include them inside this class :) Also you can make something like `private static {staticMethodName}` and then use inside another static methods `this.{staticMethodName}` 😄

Comment (jonaslagoni) [reply src/simplification/Simplifier.ts:6]: I mean I use them in tests separately. Do you suggest adding everything from the separate functions and add them as static functions inside the `Simplifier` class? 🤔

Comment (magicmatatjahu) [reply src/simplification/Simplifier.ts:6]: If it is a problem for you, you can leave it, but functions are only used inside `simplify` function

Comment (jonaslagoni) [reply src/simplification/Simplifier.ts:6]: Yes, but IMO separate files for these simplification rules are the best solution. The class file in the end would be way too bloated.

Comment (magicmatatjahu) [reply src/simplification/Simplifier.ts:6]: OK 😄

**Review by magicmatatjahu — APPROVED** [review]

**Pull request merged**